horizon/templates/local_settings.py.erb

```diff
@@ -261,7 +261,7 @@ API_RESULT_PAGE_SIZE = 20
 
 # The timezone of the server. This should correspond with the timezone
 # of your entire OpenStack installation, and hopefully be in UTC.
-TIME_ZONE = "UTC"
```

Comment:
@Ruoyu111 This looks good, I am just curious - does this handle Daylight Savings Time?